src/metabase/util.cljc

```diff
@@ -207,6 +208,25 @@
     (str (upper-case-en (subs s 0 1))
          (subs s 1))))
 
+(declare ^:private ->kebab-case-en*)
+
+(csk.macros/defconversion "kebab-case-en*" lower-case-en lower-case-en "-")
+
+(defn- wrap-csk-conversion-fn-to-handle-nil-and-namespaced-keywords
+  "Wrap a CSK defconversion function so that it handles nil and namespaced keywords, which it doesn't support out of the
+  box for whatever reason."
+  [f]
+  (fn [x]
+    (when x
+      (if (qualified-keyword? x)
+        (keyword (f (namespace x)) (f (name x)))
+        (f x)))))
+
+(def ^{:arglists '([x])} ->kebab-case-en
```

Comment:
partially taken from https://github.com/metabase/metabase/pull/30248, I use this function in `metabase.api.card` to transform field metadata